src/hooks/use-mounted-layout-effect.js

```diff
@@ -0,0 +1,23 @@
+import {
+  useEffect,
+  useLayoutEffect,
+  useRef,
+} from 'react';
+
+export default function useMountedLayoutEffect(callback, dependencies) {
```

Comment:
Rename to something similar to React lifecycle?


```suggestion
export default function useUpdatedLayoutEffect(callback, dependencies) {
```